gammapy/scripts/analysis.py

```diff
@@ -51,6 +60,124 @@ def settings(self):
         """Configuration settings for the analysis session."""
         return self.config.settings
 
+    def get_observations(self):
+        """Fetch observations from the data store according to criteria defined in the configuration."""
+        self.config.validate()
+
+        datastore_path = make_path(self.settings["observations"]["datastore"])
+        if datastore_path.is_file():
+            self.datastore = DataStore().from_file(datastore_path)
+        elif datastore_path.is_dir():
+            self.datastore = DataStore().from_dir(datastore_path)
+        else:
+            log.error("Datastore {} not found.".format(datastore_path))
+            return False
```

Comment:
Shouldn't we raise an error?
It's very easy to forget to look at the return value of this method and have a silent error.
Note that you don't even document the return value and how callers should handle it.
If you just raise an error, I think that's fine, no extra documentation needed.

Reply:
Ok.
I raise now a *`FileNotFoundError`*